cmd/sos_acl.go

```diff
@@ -467,35 +462,23 @@ var sosShowACLCmd = &cobra.Command{
 	},
 }
 
-func getGrantValue(values []string) string {
-	for i, v := range values {
-		values[i] = v[len("id="):]
-	}
-	return strings.Join(values, ", ")
-}
-
 func formatGrantKey(k string) string {
 	var res string
 	switch {
-	case k == manualRead:
+	case k == "READ":
 		res = "Read"
-	case k == manualWrite:
+	case k == "WRITE":
 		res = "Write"
-	case k == manualReadACP:
+	case k == "READ_ACP":
 		res = "Read ACP"
-	case k == manualWriteACP:
+	case k == "WRITE_ACP":
 		res = "Write ACP"
-	case k == manualFullControl:
+	case k == "FULL_CONTROL":
```

Comment:
You should Add those value at the top of the file in const. And add a comment to explain, those value come from the S3 response body.
If you do that could you edit this comment also:
https://github.com/exoscale/cli/blob/7d7502b1399731edfd610de6edd1d9be6cab4e66/cmd/sos_acl.go#L24
by explaining, those value come from the S3 response header. 

Thank you great job! 🚀

Reply:
Good point @pierre-emmanuelJ. I would also recommend that you use this form of `switch` statement:

```go
switch k {
    case SOSACLRead:
        res = "Read"
    case SOSACLWrite:
        res = "Write"
    // ...
}
```